src/logic/calculate.js

```diff
@@ -0,0 +1,129 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
+/**
+ * Given a button name and a calculator data object, return an updated
+ * calculator data object.
+ *
+ * Calculator data object contains:
+ *   total:s      the running total
+ *   next:String       the next number to be operated on with the total
+ *   operation:String  +, -, etc.
+ */
+export default function calculate(obj, buttonName) {
+  if (buttonName === 'AC') {
+    return {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
```

Comment:
- [ ] Let us refactor our syntax with ES6, kindly, use arrow function instead of using function

- [ ] Remove the commented lines in our code.

Reply:
We are asked to copy and paste this line of code from a given file and we are not asked to change anything from it that Why I left it that way.

Reply:
Okay, get it